sdk/samples/iot/README.md

```diff
@@ -602,149 +606,29 @@ Additional instructions for Azure IoT Explorer, including screenshots, are avail
-
-    On initial bootup of the device, the sample will send the Temperature Controller reported properties to the service. You will see the following in the device twin JSON.
-
-    ```json
-    "properties": {
-        "reported": {
-            "manufacturer": "Sample-Manufacturer",
-            "model": "pnp-sample-Model-123",
-            "swVersion": "1.0.0.0",
-            "osName": "Contoso",
-            "processorArchitecture": "Contoso-Arch-64bit",
-            "processorManufacturer": "Processor Manufacturer(TM)",
-            "totalStorage": 1024,
-            "totalMemory": 128,
-            "serialNumber": "ABCDEFG",
-        }
-    }
-    ```
-
-    <b>To send a device twin desired property message:</b> Select your device's Device Twin tab in the Azure IoT Explorer. Add the property targetTemperature along with a corresponding value to the desired section of the JSON. Select Save to update the twin document and send the twin message to the device.
-
-    ```json
-    "properties": {
-        "desired": {
-            "thermostat1": {
-                "targetTemperature": 34.8
-            },
-            "thermostat2": {
-                "targetTemperature": 68.5
-            }
-        }
-    }
-    ```
-
-    No other property names sent in a desired property message are supported. If any are sent, the log will report there is nothing to update.
-
-    Upon receiving a desired property message, the sample will update the twin property locally and send a reported property of the same name back to the service. This message will include a set of "ack" values: `ac` for the HTTP-like ack code, `av` for ack version of the property, and an optional `ad` for an ack description.
-
-    ```json
-    "properties": {
-        "reported": {
-            "thermostat1": {
-                "__t": "c",
-                "maxTempSinceLastReboot": 38.2,
-                "targetTemperature": {
-                    "value": 34.8,
-                    "ac": 200,
-                    "av": 27,
-                    "ad": "success"
-                }
-            },
-            "thermostat2": {
-                "__t": "c",
-                "maxTempSinceLastReboot": 69.1,
-                "targetTemperature": {
-                    "value": 68.5,
-                    "ac": 200,
-                    "av": 28,
-                    "ad": "success"
-                },
-            }
-        }
-    }
-    ```
-
-    </p>
-    </details>
-
-    <details><summary><b>Commands:</b></summary>
-    <p>
-
-    Two device commands are supported in this sample: `reboot` and `getMaxMinReport`.
+    The easiest way to interact with this sample from the service side is to use Azure IoT Explorer.  To use the sample:
 
-    - To invoke `reboot` on the Temperature Controller, select the "Default component" and go to the "Commands" tab. Find `reboot` and enter a delay payload. Select "Send command".
```

Comment:
I am.  The place to document this in my opinion is IoT Explorer itself.  My concerns are (a) if they update their UX at some point, we'll end up with stale instructions, and (b) this would need to be duplicated across all SDKs.

https://github.com/Azure/azure-iot-explorer/issues/441 tracks my request on IoT Explorer to document the "golden" example models which the PM, Rido, has already agreed with.